cnames_active.js

```diff
@@ -1004,7 +1004,7 @@ var cnames_active = {
   "itzik": "itzik.netlify.app",
   "iutdb": "xchairs.github.io/iutdb.github.io",
   "ivy": "sinistrecyborg.github.io/Ivy",
-  "iwanttocountfromone": "realcyguy.github.io/iwanttocountfromone.js",
+  "": "realcyguy.github.io/iwanttocountfromone.js",
```

Comment:
```suggestion
  "iwanttocountfromone": "realcyguy.github.io/iwanttocountfromone.js",
```
Please revert this change